aksel.nav.no/website/pages/eksempler/formsummary/demo.tsx

```diff
@@ -62,6 +61,9 @@ const Example = () => {
           <FormSummary.Value>SMS</FormSummary.Value>
         </FormSummary.Answer>
       </FormSummary.Answers>
+      <FormSummary.Footer>
```

Comment:
My OCD really wants an empty line before the footer for consistency 😁

```suggestion

      <FormSummary.Footer>
```